build/go.mod

```diff
@@ -4,8 +4,22 @@ go 1.12
 
 require (
 	github.com/go-git/go-git/v5 v5.4.2
-	github.com/mattermost/mattermost-server/v6 v6.2.1
+	github.com/google/uuid v1.3.0 // indirect
+	github.com/json-iterator/go v1.1.12 // indirect
+	github.com/klauspost/cpuid/v2 v2.0.9 // indirect
+	github.com/kr/pretty v0.3.0 // indirect
+	github.com/mattermost/go-i18n v1.11.1-0.20211013152124-5c415071e404 // indirect
+	github.com/mattermost/mattermost-server/v5 v5.39.3
```

Comment:
We came to know that one of the MM's customers PNC is using `v5.39.3` which may use this plugin so, we downgraded it.

Reply:
@avas27JTG FYI this should still work even if we compile with `6.x`. There was a breaking change in `5.37` but I'm not sure if there has been one since then

Reply:
Sure @mickmister it compiles with `6.x`
`5.37.3` is the min required version